### ISSUE TITLE
Extremely slow node traversal

### ISSUE THREAD

**Issue by 4c74356b41** [projectkudu/AzureResourceExplorer#335]
Howdy, on one tenant I'm observing extremely slow node traversal (like 1 - 1.5 minutes to expand any node). Portal\az\powershell work fine for the same tenant. I'm not sure what to provide as the evidence (I've provided the har file to support, but I'm not sure how to anonymize its contents for public usage). But the issue was confirmed by support: 2204010040004597

**Comment by balag0** [comment]
Hi, Was this recent? The last update we did was March 1.

**Comment by 4c74356b41** [comment]
no, this started in december\november, but this is when i started working on a new tenant\rpoject, so its might have been sluggish all the time there.

and it only happens for this one tenant.
